src/gan/gen_pyt/gen_config.cfg

```diff
@@ -0,0 +1,98 @@
+
+#### General configuration ####
```

Comment:
this feels like another config that was reviewed earlier in the PR... are they different in substantive ways?

Reply:
This is specific to the new pytorch version of the generator, but based on this and another config/args parsing comment above, I'm merging the two cfg files and updating to take all GAN parameters from one `config.cfg` file.